librz/include/rz_core.h

```diff
@@ -418,19 +418,24 @@ RZ_API void rz_core_item_free(RzCoreItem *ci);
 
 typedef int (*RzCoreSearchCallback)(RzCore *core, ut64 from, ut8 *buf, int len);
 
-typedef struct sym_name_t {
-	const char *pfx; // prefix for flags
-	char *name; // raw symbol name
-	char *rz_symbol_name;
-	char *libname; // name of the lib this symbol is specific to, if any
-	char *nameflag; // flag name for symbol
-	char *demname; // demangled raw symbol name
-	char *demflag; // flag name for demangled symbol
-	char *classname; // classname
-	char *classflag; // flag for classname
-	char *methname; // methods [class]::[method]
-	char *methflag; // methods flag sym.[class].[method]
-} SymName;
+/**
+ * \brief Store some display name from RzBinSymbol
+ * \see rz_core_sym_name_init
+ * \see rz_core_sym_name_fini
+ */
+typedef struct rz_sym_name_t {
+	const char *pfx; ///< prefix for flags
+	char *name; ///< raw symbol name
+	char *symbolname; ///< display symbol name
+	char *libname; ///< name of the lib this symbol is specific to, if any
+	char *nameflag; ///< flag name for symbol
+	char *demname; ///< demangled raw symbol name
+	char *demflag; ///< flag name for demangled symbol
+	char *classname; ///< classname
+	char *classflag; ///< flag for classname
+	char *methname; ///< methods [class]::[method]
+	char *methflag; ///< methods flag sym.[class].[method]
+} RzSymName;
```

Comment:
@XVilka ? WDYT? Better name maybe? It has a very specific use case but the name is very generic, I'm not 100% sure it is good as-is.

Reply:
I think, `RzBinSymNames` would be a better choice, note the *Names*

Reply:
RzCoreSymNames. It's in RzCore, not RzBin.